diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/spaces/IndentationRuleTestResources.kt

```diff
@@ -605,6 +605,29 @@ internal object IndentationRuleTestResources {
         |            2 to
         |            3))
         """.trimMargin(),
+
```

Comment:
Actually - I suggest to split resources - we don't need to have huge unit tests that have complex logic (especially when warnings are far away from the code)